README.md

```diff
@@ -1,10 +1,58 @@
 # Npgsql Entity Framework Core provider for PostgreSQL
 
-[![stable](https://img.shields.io/nuget/v/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=stable)](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL/) [![unstable](https://img.shields.io/myget/npgsql-unstable/vpre/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=unstable)](https://www.myget.org/feed/npgsql-unstable/package/nuget/Npgsql.EntityFrameworkCore.PostgreSQL) [![next patch](https://img.shields.io/myget/npgsql/v/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=next%20patch)](https://www.myget.org/feed/npgsql/package/nuget/Npgsql.EntityFrameworkCore.PostgreSQL)
-[![appveyor](https://img.shields.io/appveyor/ci/roji/npgsql-entityframeworkcore-postgresql/dev.svg?label=appveyor)](https://ci.appveyor.com/project/roji/npgsql-entityframeworkcore-postgresql) [![travis](https://img.shields.io/travis/npgsql/npgsql.svg?label=travis)](https://travis-ci.org/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL) [![gitter](https://img.shields.io/badge/gitter-join%20chat-brightgreen.svg)](https://gitter.im/npgsql/npgsql)
+[![gitter](https://img.shields.io/badge/gitter-join%20chat-brightgreen.svg)](https://gitter.im/npgsql/npgsql)
 
 Npgsql.EntityFrameworkCore.PostgreSQL is an Entity Framework Core provider built on top of
 [Npgsql](https://github.com/npgsql/npgsql). It allows you to use the EF Core O/RM with
 PostgreSQL.
 
 Documentation is available on [the Npgsql site](http://www.npgsql.org/efcore/index.html).
+
+# Build Status
+
+| Branch                               | Build Status                                     | Stable                                 | Next Patch                                  | Unstable                                                    |
+|:-------------------------------------|:-------------------------------------------------|:---------------------------------------|:--------------------------------------------|:------------------------------------------------------------|
+| [`dev`][tree-dev]                    | [![status][build-dev]][ci-dev]                   | [![stable][nuget-vpre]][nuget-npgsql]  |                                             | [![unstable][npgsql-unstable-vpre]][myget-npgsql-unstable]  |
+| [`master`][tree-master]              | [![status][build-master]][ci-master]             | [![stable][nuget-v]][nuget-npgsql]     | [![next patch][npgsql-vpre]][myget-npgsql]  |                                                             |
+| [`hotfix/2.2.6`][tree-hotfix/2.2.*]  | [![status][build-hotfix/2.2.*]][ci-hotfix/2.2.*] | [![stable][nuget-2.2.*]][nuget-npgsql] | [![next patch][npgsql-2.2.*]][myget-npgsql] | [![unstable][npgsql-unstable-2.2.*]][myget-npgsql-unstable] |
```

Comment:
It seems like a bad idea to actually embed version numbers in this way - there's no way we'll remember to update this after every single patch version...

The current badges generate the version automatically, is it not possible to have the same?

Reply:
I can get `v` and`vpre` from each feed, but not `latest patch for version X` (very disappointing).

Reply:
Hmm, OK - the previous badges with the versions were for the nuget packages, rather than for the build. We can leave those badges (along with [new Azure Devops](https://shields.io/category/build) replacing the Appveyor and Travis ones.

The complete build info as seen here can definitely be in README.md on github (that seems standard) and in another page on the docs for those interested)...